tests/integration/components/schema-chunk/validation.ts

```diff
@@ -0,0 +1,12 @@
+import { ValidationObject } from 'ember-changeset-validations';
+import { validatePresence } from 'ember-changeset-validations/validators';
+import NodeModel from 'ember-osf-web/models/node';
+
+export const nodeValidation: ValidationObject<NodeModel> = {
+    title: [
+        validatePresence(true),
+    ],
+    description: [
+        validatePresence(true),
+    ],
+};
```

Comment:
Is it necessary for this to be in a separate file? I dunno.

Reply:
Since it's not being used for any of the tests, I'll remove it here. :+1: